### ISSUE TITLE
list-wsl gives errors

### ISSUE THREAD

**Issue by samdark** [sirredbeard/wslinternals#1]
```
PS C:\src\wslinternals> .\list-wsl.ps1
You cannot call a method on a null-valued expression.
At C:\src\wslinternals\list-wsl.ps1:19 char:9
+         $distribution["Distro Version"] = ($osRelease | Where-Object  ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

id: unknown user 0
cat: /etc/wsl.conf: No such file or directory
Processing fstab with mount -a failed.

<3>WSL (17) ERROR: CreateProcessEntryCommon:370: getpwuid(0) failed 2
<3>WSL (17) ERROR: CreateProcessEntryCommon:374: getpwuid(0) failed 2
<3>WSL (17) ERROR: CreateProcessEntryCommon:577: execvpe /bin/sh failed 2
<3>WSL (17) ERROR: CreateProcessEntryCommon:586: Create process not expected to return
Processing fstab with mount -a failed.

<3>WSL (20) ERROR: CreateProcessEntryCommon:370: getpwuid(0) failed 2
<3>WSL (20) ERROR: CreateProcessEntryCommon:374: getpwuid(0) failed 2
<3>WSL (20) ERROR: CreateProcessEntryCommon:577: execvpe /bin/sh failed 2
<3>WSL (20) ERROR: CreateProcessEntryCommon:586: Create process not expected to return
Processing fstab with mount -a failed.

<3>WSL (23) ERROR: CreateProcessEntryCommon:370: getpwuid(0) failed 2
<3>WSL (23) ERROR: CreateProcessEntryCommon:374: getpwuid(0) failed 2
<3>WSL (23) ERROR: CreateProcessEntryCommon:577: execvpe /bin/sh failed 2
<3>WSL (23) ERROR: CreateProcessEntryCommon:586: Create process not expected to return

Name                Linux Distro       Distro Version            Default User systemd State     WSL
----                ------------       --------------            ------------ ------- -----     ---
docker-desktop      Docker Desktop                                              False Installed   2
Ubuntu*             Ubuntu 20.04.2 LTS 20.04.2 LTS (Focal Fossa) samdark              Installed   2
docker-desktop-data                                                                   Installed   2
```

**Comment by sirredbeard** [comment]
It seems to be failing to get info on docker-desktop-data, which might not contain an /etc/os-release file or other binaries expected, like `id`

**Comment by sirredbeard** [comment]
Fixed in #2 and release 0.0.3 https://github.com/sirredbeard/wslinternals/releases/tag/0.0.3
